lib/y2_common.pm

```diff
@@ -0,0 +1,57 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: This module provides common subroutines for YaST2 modules in graphical and text mode
+# Maintainer: Joaquín Rivera <jeriveramoya@suse.com>
+
+package y2_common;
+
+use base Exporter;
+use Exporter;
+
+use strict;
+use testapi;
+use version_utils qw(is_opensuse is_leap);
+
+our @EXPORT = qw(is_network_manager_default
+  continue_info_network_manager_default
+  accept_warning_network_manager_default
+);
+
+=head2 is_network_manager_default
+openSUSE has network manager as default except for older leap versions
+=cut
+sub is_network_manager_default {
+    return is_opensuse unless is_leap('<=15.0');
```

Comment:
I know that this peace of code works, but just to be consistent, don't you mind rewriting it as 
`return is_opensuse && !is_leap('<=15.0')`. Problem with consistency here is that we explicitly return 0 or 1 if it's not openSUSE, and undef if it is. 0 and undef both mean false, but we can be consistent here.
It's a neat pick, so feel free to leave it as it is ;)

Reply:
To be honest for me it's less readable, but that's my personal feeling about it ;)